tests/e2e/migrations/0000_oasis_3_test_e2e.up.sql

```diff
@@ -0,0 +1,285 @@
+-- Indexer state initialization for the Damask Upgrade.
```

Comment:
I was of the opinion that it was useful to have it's own tables to really create a sandbox environment for the tests. I like just having a different DB instance for this purpose, too.

Reply:
Discussed on slack: This is connected to #153. Since the contents of mainnet genesis are in our migrations files, e2e doesn't run the "normal" migrations (so they're never tested!) and has this separate copy instead. This increases the priority of #153. Once that issue is closed, we can get rid of this file.